### ISSUE TITLE
symfony/translation version requirement prevents installation in Symfony 6.0

### ISSUE THREAD

**Issue by adorbabble** [createnl/zxcvbn-bundle#8]
Hi, when trying to upgrade from Symfony 5.4 to 6.0, I get this :
```
Problem 2
    - createnl/zxcvbn-bundle 1.0.0 requires php ^7.3 -> your php version (8.0.13) does not satisfy that requirement.
    - createnl/zxcvbn-bundle 1.1.0 requires symfony/translation ^5.2 -> found symfony/translation[v5.2.0-BETA1, ..., 5.4.x-dev] but it conflicts with your root composer.json require (6.0.*).
    - Root composer.json requires createnl/zxcvbn-bundle ^1.0 -> satisfiable by createnl/zxcvbn-bundle[1.0.0, 1.1.0].
```

**Comment by AlexLisenkov** [comment]
I have created a pr https://github.com/createnl/zxcvbn-bundle/pull/9. But need to wait for https://github.com/bjeavons/zxcvbn-php/pull/60 to finish first.

**Comment by mkopinsky** [comment]
The linked PR in bjeavons/zxcvbn-php has been merged 😄

**Comment by AlexLisenkov** [comment]
Released tag 2.0.0
